libs/ts-rest/core/src/lib/dsl.ts

```diff
@@ -173,13 +190,27 @@ type ContractInstance = {
    */
   mutation: <T extends AppRouteMutation>(mutation: T) => T;
   /**
-   * Exists to allow storing a Type in the contract (at compile time only)
+   * @deprecated Please use type() instead.
+   */
+  response: <T>() => T extends null ? ContractNullType : ContractPlainType<T>;
+  /**
+   * @deprecated Please use type() instead.
    */
-  response: <T>() => T;
+  body: <T>() => T extends null ? ContractNullType : ContractPlainType<T>;
   /**
    * Exists to allow storing a Type in the contract (at compile time only)
    */
-  body: <T>() => T;
+  type: <T>() => T extends null ? ContractNullType : ContractPlainType<T>;
+  /**
+   * Define a custom response type
+   */
+  otherResponse: <T extends ContractAnyType>({
+    contentType,
+    body,
+  }: {
+    contentType: string;
+    body: T;
+  }) => ContractOtherResponse<T>;
```

Comment:
We're gonna see so many warnings in our prod code 🙈

Reply:
😆